pkg/cmd/auth/logout/logout.go

```diff
@@ -152,6 +148,8 @@ func logoutRun(opts *LogoutOptions) error {
 		return fmt.Errorf("failed to write config, authentication configuration not updated: %w", err)
 	}
 
+	isTTY := opts.IO.IsStdinTTY() && opts.IO.IsStdoutTTY()
```

Comment:
I wonder if we could also name these tty checks something more descriptive that declares intent, like `opts.IO.ShouldShowConfirmation()` (ideally with a better name). Perhaps in a separate changeset!

Reply:
I like the idea of an `IO.ShowConfirmations()`! i'll open an issue for it.